cost-analyzer/templates/cost-analyzer-ingestion-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.kubecostProductConfigs.standardDiscount }}
```

Comment:
```suggestion
{{- if (.Values.kubecostProductConfigs).standardDiscount }}
```

I believe you will need this to fix the Chart Build issue. "Only try to dereference standardDiscount if kubecostProductConfigs is defined".

Reply:
Yeah oops, just saw that. My bad.